edb/server/dbview/dbview.pyx

```diff
@@ -557,12 +558,12 @@ cdef class DatabaseConnectionView:
                 state.append({"name": sval.name, "value": jval, "type": kind})
 
         # Include the database version in the state so that we are forced
-        # to clear the config cache on dbver changes.
+        # to clear the config cache on changes.
         state.append(
-            {"name": '__dbver__', "value": dbver, "type": 'C'})
+            {"name": '__ver__', "value": ver, "type": 'C'})
```

Comment:
Where are we actually reading this?

Reply:
https://github.com/edgedb/edgedb/blob/669ba34661ad04a605b12f745fa4b92122557570/edb/server/protocol/execute.pyx#L88-L91 and https://github.com/edgedb/edgedb/blob/669ba34661ad04a605b12f745fa4b92122557570/edb/server/protocol/execute.pyx#L253-L256 are the relevant places.

It doesn't get used directly, but changes in it are used to force a state sync